### DISCUSSION TITLE
how do I update the app?  says downlevel, but no link to update, not in Google Play Store

### DISCUSSION THREAD

**Discussion by sdetweil** [mealie-recipes/mealie#3326] [General]
I have the Mealie app installed on my Android phone

says v1.1.0  consider upgrading to latest 1.3.2

**Comment by michael-genson** [comment]
In the future please use an issue template before creating an issue. You can find them here: https://github.com/mealie-recipes/mealie/issues/new/choose

Regarding your question: there is no official Mealie app, so I'm not sure what you're using or how to update it. Mealie is a self-hosted application, so you'll need to talk to the server owner about upgrading the Mealie instance.

**Comment by sdetweil** [comment]
ok, I am the server owner.

wasn't clear, says install app, find thru apps

issue template choices say if not these use empty

**Comment by michael-genson** [comment]
If you're the server owner, you can update Mealie using the latest docker image. If you can provide more information regarding how you're deploying Mealie I can provide more information, but generally it's as simple as replacing the image tag and re-creating the container.

Keep in mind any time you update Mealie you should create a backup, just in case something goes wrong.
